lineQueueClient/src/components/Signup/Signup.style.js

```diff
@@ -1,33 +1,58 @@
-import { StyleSheet } from 'react-native';
+import {StyleSheet, Dimensions} from 'react-native';
+
+let width = Dimensions.get('window').width
+let height = Dimensions.get('window').height
 
 const signupStyle = StyleSheet.create({
-  containerWhite: {
-    flex: 1,
+  contentContainer: {
+    display: 'flex',
     flexDirection: 'column',
-    justifyContent: 'center',
+    justifyContent: 'space-around',
+    alignItems: 'stretch',
+    backgroundColor: 'white',
```

Comment:
backgroundColor is `'white'` by default

Reply:
checked it - background isn't white by default, it's no-fill by default. backgroundColor should remain there.